Community-Aktivitäten/README.md

```diff
@@ -165,6 +165,46 @@ Dauer: 2 Stunden (bei Bedarf auch gerne mehr)
 
 Ausstattung: großer Screen/Beamer, Internet
 
+## Building your career: Ask anything
+:us: :de:
+
+### Collaborators
+* [JUG Schweiz]: [Ixchel Ruiz]
+* [JUG Dortmund]: [Sandra Parsick]
+* [JUG Dortmund]: [Hendrik Ebbers]
+
+### Description:
+If you're looking for career advice in software development, a good strategy is to ask your seniors. They have valuable experience and can provide insights that will help guide your professional development. Their first-hand knowledge can give you a clear understanding of the industry and its requirements. Don't hesitate to join this event with any questions or concerns you may have. By seeking the guidance and advice of our guests, you'll be better equipped to make informed decisions about your career path.
+
+### Duration : 2 hr : 1 hr Tuesday & Wednesday during the conference ( 2 Days ) 
+
+### Beschreibung:
+If you're looking for career advice in software development, a good strategy is to ask your seniors. They have valuable experience and can provide insights that will help guide your professional development. Their first-hand knowledge can give you a clear understanding of the industry and its requirements. Don't hesitate to come to this event with any questions or concerns that you may have. By seeking the guidance and advice of our guests, you'll be better equipped to make informed decisions about your career path.
```

Comment:
It is a bit unclear, what the attendee will actively do in this event. Is it just a possibility to ask questions to a panel, or can seniors from the audience get a seat in the panel himself?

Reply:
https://en.wikipedia.org/wiki/Fishbowl_(conversation) 

We will have people whose experience is relevant to the topic ( Seniors with ample experience ) but as this is a community event I think opening the interaction to the whole community is also important

Reply:
Ok so as long as there is interaction with the community other than "they listen and ask questions" I agree that we can adopt it as a "community event".